.changeset/pretty-zoos-suffer.md

```diff
@@ -0,0 +1,15 @@
+---
+"@salt-ds/lab": patch
+---
+
+Slider / RangeSlider updates
+
+Updated lab feature set and API for `Slider` and `RangeSlider`
+
+- track now support tick marks with the `showTicks` prop.
+- `restrictToMarks` prop will snap the value to the closest mark.
+- `decimalPlaces` prop specifies the number of decimal places for the value.
+- `constrainLabelPosition` will ensure that mark labels remain within the boundary of the track.
+- updated slider examples and docs
```

Comment:
This shouldn't be part of a changeset, since it's not a change to the actual component